packages/frontend/pages/test.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+//import MessageBoard from '../components/MessageBoard'
+import VideoBoard from '../components/Video/VideoBoard'
+//import ImageBoard from '../components/ImageBoard'
+
+export default function test() {
+  return (
+    <>
+      <div>
+        <VideoBoard />
+      </div>
+    </>
+  )
+}
```

Comment:
You committed your test file

Reply:
Otherwise looks good though

Reply:
oh damn, didnt notice that, sorry for that